### ISSUE TITLE
Add resource limits to Ansible operator scaffolding

### ISSUE THREAD

**Issue by asmacdo** [operator-framework/operator-sdk#3573]
The Helm operator scaffolds resource limits in the manager manifest:
https://github.com/operator-framework/operator-sdk/blob/24d73f782af630f26d3600530fb9b46e6878fc57/internal/plugins/helm/v1/scaffolds/internal/templates/config/manager/manager.go#L78-L84

However these resource limits were too restrictive for ansible operators. Since v0.19 scaffolded Ansible operators do not have resource limits, I am considering this a new feature that can be added post 1.0 release.

**Comment by camilamacedo86** [comment]
It is important because common attacks take advantage of it. See; https://cheatsheetseries.owasp.org/cheatsheets/Docker_Security_Cheat_Sheet.html#rule-7-limit-resources-memory-cpu-file-descriptors-processes-restarts

PS. The default values are not enough for Ansible. So, the complexity here is to find what values should be appropriated.

**Comment by asmacdo** [comment]
Some research is needed to determine what these default values should be. We have deprioritized for now-- please feel free to chime in or open a PR with suggested defaults.

**Comment by camilamacedo86** [comment]
@asmacdo, @fabianvf, @jmrodri  shows like we need to be priorized this one. 
I started to work on some POC tests to address this need. 

We need to have default values that work for our samples, note that users will be notified that they ought to optimize the values based on their projects.   See: https://github.com/operator-framework/operator-sdk/pull/5330/files#diff-063b9bbaede872aca0d6afe93d010d359c86cbc7ad4c1eab8652d8a5a46f6158R50-R51

It was also added in the docs:

> Optimize manager resource values in config/manager/manager.yaml according to project requirements. It is recommended to define resources limits in order to follow good practices and for security reasons. More info: Managing Resources for Containers and Docker Security Cheat Sheet

https://sdk.operatorframework.io/docs/best-practices/common-recommendation/

And then, see: https://github.com/operator-framework/api/issues/172
